lib/ansible/modules/service.py

```diff
@@ -1010,8 +1017,18 @@ def get_service_tools(self):
 
         self.sysrc_cmd = self.module.get_bin_path('sysrc')
 
+    def run_service_cmd(self, *, status: str) -> t.Tuple[int, str, str]:
```

Comment:
Why didn't you just accept the suggestion? (https://github.com/ansible/ansible/pull/85190#discussion_r2107703322)

That import isn't needed for this. We're on a modern enough minimum Python:
```suggestion
    def run_service_cmd(self, *, status: str) -> tuple[int, str, str]:
```